content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -196,6 +196,91 @@ add a brief description in the :guilabel:`Description` field. Click :guilabel:`F
 the :guilabel:`Rule Configuration` wizard. Now, the new rule is active and devices can connect to
 the Windows virtual :abbr:`IoT (Internet of Things)` box.
 
+Worldline exception
+~~~~~~~~~~~~~~~~~~~
+
+Worldline is a payment terminal that can be connected to Odoo's point of sale system. It allows for
+a comprehensive and fluid payment experience for customers. Worldline is available in Benelux
+(coalition of Netherlands, Belgium, Luxembourg). When using the Windows IoT server to connect the
+Worldline payment terminal it is necessary to create an exception in the Windows firewall so that
+a connection can be made between the Odoo database/:abbr:`IoT (Internet of Things)` box and
+Worldline.
+
+.. seealso::
+   :doc:`../../../sales/point_of_sale/payment_methods/terminals/worldline`
+
+To create the exception, first, open the Windows Defender Firewall on the Windows machine. This can
+be accomplished by going to the :guilabel:`Search bar` and typing in `windows defender`. In the left
```

Comment:
not that easy infortunately: first it's "Windows Defender Firewall" app
![image](https://github.com/odoo/documentation/assets/60775325/044018ee-2858-4a05-9b8b-b2e72493c7ea)
then you have to click "Advanced settings" on the left pane
![image](https://github.com/odoo/documentation/assets/60775325/80c4dbed-8c5e-415f-bd67-39f20e4314fe)